include/zephyr/usb/usb_ch9.h

```diff
@@ -75,7 +75,7 @@ struct usb_setup_packet {
  * @brief Check if request transfer direction is to host.
  *
  * @param setup Pointer to USB Setup packet
- * @return true If transfer direction is to host
+ * @retval true If transfer direction is to host
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/pull/97640/files#diff-d0da566cd07713fa9d0d6929f3ec2bc6a06473bbe05ddbd281eba2d382292f06R116-R119

perhaps drop the ch9 changes altogether, as I'll take care of this in the other PR.

Reply:
oh wow, cool! then maybe we do need to agree on the guidelines first though :)